packages/support/resources/js/sortable.js

```diff
@@ -11,6 +11,7 @@ export default (Alpine) => {
         }
 
         el.sortable = Sortable.create(el, {
+            group: el.getAttribute('x-sortable-group'),
```

Comment:
Can this not just be `[x-sortable-group]` like the others?

Reply:
and what happens if the attribute is not defined

Reply:
If not defined it gets null -> behaves like now.

Example why you need a value.

you have 4 (2 groups) lists and you want only to move items inside groups.

```
-------       -------
|     |       |     |
|     |  <->  |     |
|     |       |     |
-------       -------

-------       -------
|     |       |     |
|     |  <->  |     |
|     |       |     |
-------       -------
```